@LinProp/LinProp.m

```diff
@@ -431,161 +431,347 @@ function display(obj)
             y = LinProp.Convert2LinProp(xm);
         end
         function C = subsasgn(A, S, B)
-            switch S(1).type
-                case '()'
-                    A = LinProp(A);
-                    B = LinProp(B);
-                    if A.IsComplex && ~B.IsComplex
-                        B = complex(B);
-                    end
-                    if ~A.IsComplex && B.IsComplex
-                        A = complex(A);
-                    end
-                    ni = numel(S.subs);
-                    na = ndims(A);
-%                    nb = ndims(B);
-                    if ((ni <= 1) || (ni >= na))
-                        if ni == 0
-                            C = A;
+        %SUBSASGN Subscripted assignment.
+        %   A(I) = B assigns the values of B into the elements of A specified by
+        %   the subscript vector I.  B must have the same number of elements as I
+        %   or be a scalar. 
+        %
+        %   A(I,J) = B assigns the values of B into the elements of the rectangular
+        %   submatrix of A specified by the subscript vectors I and J.  A colon used as
+        %   a subscript, as in A(I,:) = B, indicates all columns of those rows
+        %   indicated by vector I. Similarly, A(:,J) = B means all rows of columns J.
+        %
+        %   A(I,J,K,...) = B assigns the values of B to the submatrix of A specified
+        %   by the subscript vectors I, J, K, etc. A colon used as a subscript, as in
+        %   A(I,:,K) = B, indicates the entire dimension. 
+        %
+        %   For both A(I,J) = B and the more general multi-dimensional 
+        %   A(I,J,K,...) = B, B must be LENGTH(I)-by-LENGTH(J)-by-LENGTH(K)-... , or
+        %   be shiftable to that size by adding or removing singleton dimensions, or
+        %   contain a scalar, in which case its value is replicated to form a matrix
+        %   of that size.
+        
+            if strcmp('.', {S.type})
+                error('Dot indexing is not supported for variables of this type.');   %TODO: This was allowed in  V2.4.7 of the uncLib MATLAB wrapper - but did not serve any function, as far as Dion can see.
```

Comment:
Dot indexing is not needed. And it was not working in V2.4.7.
So we can remove the TODO comment here.